microsite/sidebars.json

```diff
@@ -328,6 +328,7 @@
       "deployment/index",
       "deployment/scaling",
       "deployment/docker",
+      "deployment/flightcontrol",
```

Comment:
probably move this one two steps down :) sorry but that feels like a reasonably ordered list to scan through as an average reader